app/src/main/java/pl/hypeapp/wykopolka/model/PendingUser.java

```diff
@@ -35,6 +38,12 @@
     private String postal;
     @JsonProperty("isRegistered")
     private boolean isRegistered;
+    @JsonProperty("wishId")
```

Comment:
I am not huge fan the AndroidAnnotations. This solution make many limits. I propose meet the Gson library. This library provide simple mechanism to parse and deserialize JSON file, excellent works with another library like Retrofit.